### ISSUE TITLE
running_the_pipeline: clarify how pairing of FASTQ files is expected

### ISSUE THREAD

**Issue by kevinrue** [sims-lab/capture-c#30]
https://ccanalyser.readthedocs.io/en/latest/running_the_pipeline.html#step-3-copy-or-link-fastq-files-into-the-working-directory

It looks likes this pattern is key here:
https://github.com/sims-lab/capture-c/blob/master/ccanalyser/pipeline/pipeline.py#L191

```
"ccanalyser_preprocessing/deduplicated/(.*)_[12].fastq(?:.gz)?"
```

The docs should indicate that pairing is expected as, for instance:
- sample1_1.fastq.gz
- sample1_2.fastq.gz
- sample2_1.fastq
- sample2_2.fastq

**Comment by kevinrue** [comment]
Note that there is a related but slightly patttern here:
https://github.com/sims-lab/capture-c/blob/master/ccanalyser/pipeline/pipeline.py#L88

Make sure that all the FASTQ patterns behaves in a coordinated manner

**Comment by kevinrue** [comment]
Also, I'm a bit worried about the current example, which lacks accompanying explanation

```
* samplename_R1.fastq.gz
* samplename_1.fastq.gz
* samplename_R1.fastq
* samplename_1.fastq
```

which, to me, looks like all 4 files relate to the same sample "samplename", with no pairing between "R1/R2" and "_1.fastq/_2.fastq".

**Comment by kevinrue** [comment]
In 6112ca122b2b0e380fd4b0669c265896ff895510, I'm offering an alternative example of file setup for pairing, let me know if I got it right @alsmith151 

```
* samplename1_R1.fastq.gz
* samplename1_R2.fastq.gz
* samplename2_1.fastq
* samplename2_2.fastq
```

**Comment by alsmith151** [comment]
I agree with this alteration. It would also be worth adding in the grouping pattern for replicates e.g. samplename_rep_R1.fastq.gz

**Comment by alsmith151** [comment]
I've probably commented in the wrong place on 6112ca1

**Comment by kevinrue** [comment]
I've replied to your comment. I'm confused. We should have a call.

**Comment by alsmith151** [comment]
Probably a good idea. What you've got is correct I'm apparently just bad at giving examples.

Basically the pattern I use is:

SAMPLE_CONDITION_REPLICATE_(1|2).fastq.gz


________________________________

**Comment by alsmith151** [comment]
Merged #31
